torch/onnx/symbolic_opset9.py

```diff
@@ -728,7 +729,7 @@ def _reduce_op_symbolic(onnx_op_name, allow_multi_dim_support=True):
     @_beartype.beartype
     def symbolic(g, self, dim=None, keepdim=None):
         self = _maybe_cast_reduce_op_input(g, self)
-        if dim is None:
+        if dim is None or not dim:
```

Comment:
Isn't `not dim` enough? Although I am not so sure why we would get boolean dim from aten::norm.
```suggestion
        if not dim:
```

Reply:
Dim can be an empty tuple. I should add a comment.

Reply:
Changed to test for empty tuple